paper/paper.md

```diff
@@ -32,82 +32,125 @@ affiliations:
-One interesting observation is that the first ~10 minutes of the *cerebra* timecourse appear flat, that is, no vcfs are processed. This can be attributed to the genome interval tree construction phase. After the tree is built, files are 
-processed in a near-linear manner. Also of note is that *cerebra*'s search operations take advantage of multiprocessing.
+One interesting observation is that the first ~10 minutes of the *cerebra* timecourse appear flat, that is, no vcfs are processed [OBB: Is it possible to provide pre-built indices that take <1m to load?]. 
+This can be attributed to the genome interval tree construction phase. 
+After the tree is built, files are 
+processed in a near-linear manner. 
+Also of note is that *cerebra*'s search operations take advantage of multiprocessing.
 Thus *cerebra* should scale better to high-memory machines with more cores, though it has been designed to run on everyday hardware. 
 
 [todo: what else can i add for validation?]
 
-# Conclusion
-We believe our tool satisfies an unmet need in the bioinformatics community. Fast and accurate peptide-level summarizing
-of variants in a sequencing experiment is often crucial to understanding the underlying biology of an experimental
-system. As sequencing costs continue to drop, large-scale variant calling will become more accessible to all members
-of the community, and summary tools like *cerebra* will become more important. *cerebra* is fast and accurate and is one of the only tools that fills this niche. It offers the advantage of a single, easy-to-interpret output file, making downstream analysis accessible to non-bioinformatically inclined members of the community.
+## Conclusions
+
+Our tool satisfies an unmet need in the bioinformatics community. 
+Fast and accurate peptide-level summarizing of variants in a sequencing experiment is often crucial to understanding the underlying biology of an experimental system. 
+As sequencing costs continue to drop, large-scale variant calling will become more accessible to all members of the community, and summary tools like *cerebra* will become more important. 
+*cerebra* is fast and accurate and is one of the only tools that fills this niche. 
+It offers the advantage of a single, easy-to-interpret output file, making downstream analysis accessible to non-bioinformatically inclined members of the community.
+
+## Acknowledgments
 
-# Acknowledgments
 Funding for this work was provided by the [Chan Zuckerberg Biohub](https://www.czbiohub.org/). The authors would like
 to thank Ashley Maynard, Angela Pisco and Daniel Le for helpful discussions and support.
 
-# Correspondence
+## Correspondence
+
 Please contact `lincoln.harris@czbiohub.org`
 
-# Code
+## Code
+
 *cerebra* is written in python. Code and detailed installation instructions can be found at https://github.com/czbiohub/cerebra. In addition *cerebra* can be found on [PyPi](https://pypi.org/project/cerebra/).
 
-# References
+## References
+
 [todo: add?]

```

Comment:
do i need to add these into `paper.md`?

Reply:
ok i think i did this in accordance to [JOSS submission guidelines](https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography). looks much better now. check out the [latest version](https://github.com/czbiohub/cerebra/blob/master/paper/paper.md) of the code